Cargo.lock

```diff
@@ -1541,7 +1541,7 @@ dependencies = [
  "futures",
  "fvm_ipld_blockstore 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
  "fvm_ipld_car 0.6.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "fvm_ipld_encoding 0.3.2 (registry+https://github.com/rust-lang/crates.io-index)",
+ "fvm_ipld_encoding 0.3.2",
```

Comment:
what does (registry+https://github.com/rust-lang/crates.io-index)", mean and do

Reply:
I have no idea, I don't question the lock file.